pkg/injector/annotations/annotations.go

```diff
@@ -47,6 +47,7 @@ const (
 	KeyReadinessProbePeriodSeconds      = "dapr.io/sidecar-readiness-probe-period-seconds"
 	KeyReadinessProbeThreshold          = "dapr.io/sidecar-readiness-probe-threshold"
 	KeySidecarImage                     = "dapr.io/sidecar-image"
+	KeySidecarInjected                  = "dapr.io/sidecar-injected"
```

Comment:
Since this is a label and not an annotation, can you move it outside of the `annotations` package? It's probably ok in `pkg/injector/sidecar/consts.go`. I'd also change the name to `SidecarInjectedLabel` or something like that